docs/vehicle/optioncodes.md

```diff
@@ -277,9 +277,9 @@ return a generic set of codes related to a Model 3.
 | RF3G | Model 3 Glass Roof | | 
 | RFBK | Black Roof | | 
 | RFBC | Body Color Roof | Roof | 
-| RFFG | Glass Roof | | 
-| RFP0 | All Glass Panoramic Roof | | 
```

Comment:
@timdorr note the right value is `RFPO` and not `RFP0`

https://teslascout.com/view.php?vin=5YJSA7H48FF087307#container